tests/system/small/test_dataframe.py

```diff
@@ -4657,6 +4657,17 @@ def test_to_gbq_and_create_dataset(session, scalars_df_index, dataset_id_not_cre
     assert not loaded_scalars_df_index.empty
 
 
+def test_to_gbq_table_labels(scalars_df_index):
+    destination_table = "bigframes-dev.bigframes_tests_sys.table_labels"
```

Comment:
Ideally the test should not be hard coupled with a project/dataset. Do we have to use a fixed table name? I think the test would still serve its purpose if we did `to_gbq`  without an explicit table arg and asserted labels on the returned anonymous table.

Reply:
done in https://github.com/googleapis/python-bigquery-dataframes/pull/968!